modules/core/src/viewports/globe-viewport.ts

```diff
@@ -58,6 +58,10 @@ export type GlobeViewportOptions = {
   nearZMultiplier?: number;
   /** Scaler for the far plane, 1 unit equals to the distance from the camera to the edge of the screen. Default `1` */
   farZMultiplier?: number;
+  /** Optionally override the near plane position. `nearZMultiplier` is ignored if `nearZ` is supplied. */
```

Comment:
A bit nervous about having these overrides, especially as `nearZ` and `nearZMultiplier` do the same thing. Should we issue a warning if both are supplied?

Reply:
I agree this can be confusing (and wasn't an easy thing to catch when I was first learning it). FWIW, web-mercator-viewport has had these same overrides for a long time.

A warning would be nice.. the multipliers have defaults assigned, so we'd just need to assign those after checking the user-supplied values (or else, they'd always print).

Reply:
This does look confusing and somewhat intimidating. I think these props were originally intended as a semi-undocumented escape hatch but now I seem to see these being "touched" in every PR...

If not a coincidence, might be good to audit this API with an eye towards making it as fail proof as possible for new (and old) users.

Reply:
@ibgreen FWIW I was not changing them in [my PR](https://github.com/visgl/deck.gl/pull/9293), that was just due to github getting confused. See updated PR